test/retries.js

```diff
@@ -46,9 +47,19 @@ module.exports = {
             res.end();
         });
 
-        ask({ port : server.port, maxRetries : 1 }, function(error) {
+        ask({ port : server.port, maxRetries : 1, requestId : REQUEST_ID }, function(error) {
             assert.strictEqual(error.code, Asker.Error.CODES.RETRIES_LIMIT_EXCEEDED, 'retries limit exceeded');
 
+            assert.ok(
+                (new RegExp([
+                    'Retries limit {LIMIT:1} exceeded for request ',
+                    REQUEST_ID,
+                    ' in \\d+~\\d+ ms http://localhost:',
+                    server.port,
+                    '/'
+                ].join(''))).test(error.message),
```

Comment:
avoid joining string arrays, it's less effective than concatenation of strings for about last 2 years ;)